src/Classes/Entities/StageEntity.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Portal\Entities;
+
+class StageEntity extends ValidationEntity implements \JsonSerializable
+{
+    protected $id;
+    protected $title;
+    protected $order;
+
+    public function __construct(string $id = null, string $title = null, string $order = null)
+    {
+        $this->id = ($this->id ?? $id);
```

Comment:
is the null coalescing operator really necessary lol? Happy for it to stay of-course. Just takes a few more looks to understand whats happening.

Reply:
Yes, we think it is because we will be using this entity in the same way as other entities in the application that have this operator.